### ISSUE TITLE
Enum Treated as Empty Class

### ISSUE THREAD

**Issue by bubba-h57** [kalessil/phpinspectionsea#1831]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 8.1                      |

Current behaviour
--
![image](https://user-images.githubusercontent.com/603630/172187091-4e3ba6fc-4445-4010-b151-c08dab975d5f.png)

Expected behaviour
--
Enumerations are not allowed to have properties at all and setting dynamic properties is not allowed. Neither do we expect the majority of basic enumerations will contain methods. 

Environment details
---
![Snag_c09b11](https://user-images.githubusercontent.com/603630/172187826-5a13cf71-c9be-4ed3-ad66-d9aca671327a.png)

Information from Help - About dialog

**Comment by jrcii** [comment]
I've been running into this for a while. It's been 6 months since the bug was reported, maybe we could get it on the roadmap. Thanks guys.

**Comment by jrcii** [comment]
For now I add to all enums:

```php
/** @noinspection EmptyClassInspection https://github.com/kalessil/phpinspectionsea/issues/1831 */
```

**Comment by MktpStar** [comment]
+1

**Comment by ea-inspections-team** [comment]
Thank you for your patience, everyone. We are working on a fix.

**Comment by ea-inspections-team** [comment]
Hi everyone, we pushed a fix and include it to the next release.
